source/utilities.tex

```diff
@@ -19886,6 +19886,30 @@
 For floating-point types this field specifies the formatting precision.
 For string types it specifies how many characters will be used from the string.
 
+\pnum
+When the \tcode{L} option is used, the form used for the conversion
+is called the \defn{locale-specific} form.
```

Comment:
This just puts "local-specific" into the index. I think we want to define "locale-specific form" and use `\defnadj{locale-specific}{form}`

Reply:
yes, much better.  Thanks!